spec/Body/Body.vspec

```diff
@@ -112,15 +112,11 @@ Windshield.Wiping.Intensity:
 
 Windshield.Wiping.System:
   type: branch
-#  instances: [Primary, Secondary]
   description: Signals to control behavior of wipers in detail.
                By default VSS expects only one instance.
   comment:     These signals are typically not directly available to the user/driver of the vehicle.
-               A vehicle may have multiple mechanically disconnected wiper systems serving the same windshield.
-               If that is the case for a vehicle it is recommended to add
-               instance declarations for Primary and Secondary, possibly in a layer.
-               Primary then refers to the primary wiper system for the windshield.
-               Secondary then refers to the secondary wiper system for the same windshield.
+               The overlay layers/extensions/dual_wiper_systems.vspec can be used to modify this branch
```

Comment:
We can discuss when/how to refer to layers from "standard" VSS. For layers covering a particular topic like the layer in this PR it might be beneficial to mention it in a comment in standard VSS. For more general profiles/layers like the one for motorbike it might be more difficult to find an obvious location for a comment.